### PR TITLE
Node 8.8.0-pre

### PR DESCRIPTION
**Pull request by zliu41**

### CODE REVIEW

File: cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs [hunk 1]
```diff
@@ -1161,28 +1169,86 @@ instance
...
+          [ "kind" .= String "CollectError"
+          , "error" .= String "NoCostModel"
+          , "language" .= toJSON lang
+          ]
+      Alonzo.BadTranslation err ->
+        object
+          [ "kind" .= String "PlutusTranslationError"
+          , "error" .= case err of
+              Babbage.AlonzoContextError _alonzoCtxErr ->
+                String "???"
```
Comment (zliu41) [inline cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs:1204]: @lehins

File: cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs [hunk 2]
```diff
@@ -1161,28 +1169,86 @@ instance
...
+          [ "kind" .= String "CollectError"
+          , "error" .= String "NoCostModel"
+          , "language" .= toJSON lang
+          ]
+      Alonzo.BadTranslation err ->
+        object
+          [ "kind" .= String "PlutusTranslationError"
+          , "error" .= case err of
+              Conway.BabbageContextError _babbageCtxErr ->
+                String "???"
```
Comment (zliu41) [inline cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs:1249]: @lehins

File: cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs [hunk 3]
```diff
@@ -1161,28 +1169,86 @@ instance
...
+              Conway.CertificateNotSupported _cert ->
+                String "???"
```
Comment (zliu41) [inline cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs:1251]: @lehins

File: cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs (hunk 2 above)
Comment (lehins) [reply cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs:1249]: Use instances in this PR instead for `CollectEror`

**Comment by zliu41** [comment]
https://github.com/IntersectMBO/cardano-node/pull/5648

**Pull request closed without merging**